suites/fs/multiclient/mount/fuse.yaml

```diff
@@ -0,0 +1 @@
+../../basic/mount/fuse.yaml
```

Comment:
I'm not really comfortable referencing down into parallel trees -- people generally aren't prepared for that and we'll miss it in future reviews. What's the benefit of sharing these besides removing a single task line that actually does vary across some tests?